### ISSUE TITLE
orient.getGraph() Regretion 2.1-rc2

### ISSUE THREAD

**Issue by Bretto** [orientechnologies/orientdb#4255]
I am creating a dynamic hook 
```
alter class V superclass OTriggered
ALTER CLASS V CUSTOM onAfterUpdate=setUpdated
```
the function: __setUpdated__
```
var gdb = orient.getGraph();
gdb.command('sql', 'update ' + doc.getIdentity() + ' set updated= "' +
            new java.text.SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(new java.util.Date()) + '"');
gdb.commit();
```
the error:

```
{
    "errors": [
        {
            "code": 400,
            "reason": "Bad request",
            "content": "Error on parsing script at position #0: Error on execution of the script\nScript: test\n------^\nsun.org.mozilla.javascript.internal.EcmaError: TypeError: Cannot find function getGraph in object com.orientechnologies.orient.core.command.script.OScriptOrientWrapper@218261b7. (<Unknown source>#2) in <Unknown source> at line number 2\nTypeError: Cannot find function getGraph in object com.orientechnologies.orient.core.command.script.OScriptOrientWrapper@218261b7. (<Unknown source>#2)"
        }
    ]
}
```

__Cannot find function getGraph in object com.orientechnologies.orient.core.command.script.OScriptOrientWrapper__

On an other note, I would like to write the function in the console but due to the quotes I haven't been able to write it. Could someone show me how to write this function with the proper way of escaping the quotes ?

**Comment by luigidellaquila** [comment]
Hi @Bretto 

which OrientDB version are you using? AFAIK this problem was fixed in latest release

Luigi

**Comment by Bretto** [comment]
I am using 2.1-rc2, how do you escape the quotes to write this hook in the console ?

**Comment by lvca** [comment]
@Bretto Did you 2.1.2?

**Comment by lgaquino** [comment]
About the "Cannot find function getGraph in object com.orientechnologies.orient.core.command.script.OScriptOrientWrapper", I'm getting this on version 2.1.2 .

I have a JS function that runs fine in. Studio, but not in Eclipse. It returns the following problem:
```
EcmaError: TypeError: Cannot find function getGraph in object com.orientechnologies.orient.core.command.script.OScriptOrientWrapper@183058.
```
I've looked around other issues and discussions, and it seems people solved this by simply having that following handler in orientdb-server-config.xml. But that does nothing for me..
```xml
        <handler class="com.orientechnologies.orient.graph.handler.OGraphServerHandler">
            <parameters>
                <parameter name="enabled" value="true"/>
                <parameter name="graph.pool.max" value="50"/>
            </parameters>
        </handler>
```

What I did notice was that returning the 'orient' object in studio, I get the **OScriptGraphOrientWrapper** class. While doing the same on Eclipse gives me the **OScriptOrientWrapper** class. And the latter doesn't have the getGraph() method.

How should I go around this?
